manifests/init.pp

```diff
@@ -94,6 +104,22 @@
       ip_prefixlen => $mesh_ipv6_prefixlen;
   }
 
+  if $fastd_igw_enable {
+    Class['ffnord::ntp'] -> ffnord::fastd { "fastd_igw_${mesh_code}":
```

Comment:
It is not entirely obvious why this should be applied after the ntp service is setup.